manager.spec.yaml

```diff
@@ -545,6 +545,9 @@ paths:
         - $ref: '#/components/parameters/rule_presence'
         - $ref: '#/components/parameters/show_advisories'
         - $ref: '#/components/parameters/advisory'
+        - $ref: '#/components/parameters/sap_sids'
```

Comment:
Can you show some example using `sap_sids` and `sap_system` for these APIs? For me, it doesn't make sense for these 2 APIs.

Reply:
I'm also can't see much sense, but it was requested by @psegedy and @tkasparek

Reply:
Ok, let's wait on their answer

Reply:
sorry my bad, I might have been looking at /affected_systems spec instead of /{inventory id}/cves, `rule_key` is probably not needed too

Reply:
no problem :)